config/routes.rb

```diff
@@ -6,10 +6,10 @@
   get 'table-of-contents', to: redirect('courses')
 
   resources :sections, except: [:show] do
-    resources :lessons, except: [:new, :create, :show]
+    resources :lessons, only: [:index]
   end
-  resources :lessons, except: [:edit]
+  resources :lessons
```

Comment:
The `index` and `show` routes are needed for both nested and un-nested URLs.

The un-nested `index` route is needed for displaying the list of lessons when searching and when viewing deleted lessons.

The un-nested `show` route is needed because an author is redirected to the un-nested show view after editing/updating a lesson (now that editing and updating have been un-nested as well). Within the app, lessons are still linked to the nested route. This is simply for authors. Also, if an author is redirected elsewhere, it would require re-writing a number of tests. Not a big deal, but this way seemed cleaner.